### PR TITLE
Add docs on multiline comments

### PR DESCRIPTION
**Pull request by gem-neo4j**

### CODE REVIEW

**Comment by gem-neo4j** [comment]
[GQL compliance - multiline comment](https://trello.com/c/6YVujnMs/931-gql-compliance-multiline-comment)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-893.surge.sh

Updated pages:
- [Comments](https://neo4j-docs-cypher-893.surge.sh/cypher-manual/5/syntax/comments)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-893.surge.sh

Updated pages:
- [Comments](https://neo4j-docs-cypher-893.surge.sh/cypher-manual/5/syntax/comments)

**Review by loveleif — APPROVED** [review]

File: modules/ROOT/pages/syntax/comments.adoc [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 [[cypher-comments]]
 = Comments
 
-This page describes how how to use comments in Cypher.
+This page describes how to use comments in Cypher.
 
-A comment begin with double slash (`//`) and continue to the end of the line.
+A single line comment begins with double slash (`//`) and continues to the end of the line.
+A multi-line comment begins with a slash and asterisk (`/&#42;`) and continues until the closing of the comment, which is done with an asterisk and a slash ('&#42;/'
+)
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/syntax/comments.adoc]:
```suggestion
A multi-line comment begins with a slash and asterisk (`/&#42;`) and continues until it ends with an asterisk and a slash ('&#42;/').
```

File: modules/ROOT/pages/syntax/comments.adoc [hunk 2]
```diff
@@ -22,6 +24,14 @@ MATCH (n)
 RETURN n
 ----
 
+[source, cypher, indent=0]
+----
+MATCH (n) /* This is a multi line comment,
+the comment is still going on this line as well
+but we are closing it on this line */
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/syntax/comments.adoc]:
```suggestion
the comment continues on this line
but it ends on this line. */
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-893.surge.sh

Updated pages:
- [Comments](https://neo4j-docs-cypher-893.surge.sh/cypher-manual/5/syntax/comments)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-893.surge.sh

Updated pages:
- [Comments](https://neo4j-docs-cypher-893.surge.sh/cypher-manual/5/syntax/comments)

**Review by rsill-neo4j — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**